modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -106,6 +106,26 @@ Your username must be at least three characters and contain only the following:
 [NOTE]
 You cannot create a new account within an existing organization without an email invitation.
 
+== Sign in to Anypoint Platform
+
+. Navigate to the Anypoint Platform signin screen.
+. Enter your username and password.
+. Click *Sign in*.
+
+[NOTE]
+If you try to sign in and Anypoint Platform says your username and password are incorrect when you have entered the correct credentials, your organization may use an xref:access-management::external-identity.adoc[external identity provider] to initiate the sign-in process. You can either use your external identity provider to sign you in, or you can xref:access-management::managing-your-account.adoc#use-CustomDomainLogin[enter your organization's domain name] prior to entering your credentials. 
+
+[[use-CustomDomainLogin]]
+== Sign in to Anypoint Platform using Single Sign-on
+
+If your organization uses xref:access-management::external-identity.adoc[service-provider initiated Single Sign-on], you must provide your organization's domain name as part of the signin process.
```

Comment:
I would just make it Single Sign-On in general, as this is valid for the identity provider-initiated SSO case too,
```suggestion
If your organization uses xref:access-management::external-identity.adoc[Single Sign-on],
```

Reply:
```suggestion
If your organization uses xref:access-management::external-identity.adoc[Single Sign-on], you must provide your organization's domain name as part of the sign-in process.
```